### PR TITLE
Functions Worksheet

### PR DESCRIPTION
**Pull request by spitsfire**
Tried a paragraph challenge for the code questions. couldn't test them during `learn preview`, so I'm not sure if the answers show up upon submission

### CODE REVIEW

File: 01-js-week-1/functions-worksheet.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+<!--optional, checkpoints only-->
+##### !rubric
+
+##### !end-rubric
+
+<!--optional-->
+##### !explanation
+
+```javascript
+const sayNum = function (number) {
```
Comment (CheezItMan) [inline 01-js-week-1/functions-worksheet.md:104]: Not sure why, but learn seems to strip the syntax highlighting on this explanation.

**Review by CheezItMan — APPROVED** [review]
Nice work on the functions worksheet.   The 1st class functions lesson looks good, do we have a video?  

Do we need one?

File: 01-js-week-1/functions-worksheet.md (hunk 1 above)
Comment (spitsfire) [reply 01-js-week-1/functions-worksheet.md:104]: ok I'll look into it and see if I made a syntax error

Comment (dHelmgren) [reply 01-js-week-1/functions-worksheet.md:104]: This looks fixed from my end. Closing out the PR and linking!

**Pull request merged**